pkgs/tools/security/creddump/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub, python27, python27Packages }:
```

Comment:
Please use `python2` and `python2Packages`.

Reply:
Fixed